source/javascripts/integration_usage.js

```diff
@@ -0,0 +1,21 @@
+
+
+async function setUsage() {
+    const domain = document.querySelector("#integration_usage_script").getAttribute("domain")
+    const target_element = document.querySelector("#integration_usage")
+
+    const res = await fetch("https://analytics.home-assistant.io/data.json")
```

Comment:
Why resolve this in the browser end? We can also download this during build time and add it as a data file to Jekyll?

Reply:
it will do that, still testing